alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -348,7 +348,13 @@ protected function getLivePackagerUrl($entryServerNode, $streamFormat = null)
 		$serverNode = $entryServerNode->serverNode;
 		$protocol = $this->getDynamicAttributes()->getMediaProtocol();
 		$segmentDuration = $this->getDynamicAttributes()->getEntry()->getSegmentDuration();
-		
+
+		/* @var $entry LiveStreamEntry */
+		$entry = $this->getDynamicAttributes()->getEntry();
```

Comment:
you can move this line before setting $segmentDuration and then use it for the segment duration instead of retrieving it twice

Reply:
Done